src/constants/networks.ts

```diff
@@ -48,6 +51,7 @@ export const NETWORKS_INFO_CONFIG: NETWORKS_INFO_CONFIG_TYPE = {
   [ChainId.ZKSYNC]: zksync,
   [ChainId.LINEA_TESTNET]: lineaTestnet,
   [ChainId.SOLANA]: solana,
+  [ChainId.SOLANA_TESTNET]: solanaTestnet,
```

Comment:
lol

Reply:
what would be the usecase

Reply:
campaign claim reward, seperate network.
backend dont need  to update frequently 🙏